docs/src/main/sphinx/functions/table.rst

```diff
@@ -69,3 +69,46 @@ You cannot mix the argument conventions in one invocation.
 
 All arguments must be constant expressions, and they can be of any SQL type,
 which is compatible with the declared argument type.
+
+.. _table-functions-pass-through:
+
+Pass-through function
+---------------------
+
+Trino uses a table function to support query pass-through, which allows you to
+query an underlying database through its connector. Pass-through queries require
+syntax native to the underlying database, because the full query is pushed down
+and performed in-place. This can be useful for accessing native features which
+are not implemented in Trino.
+
+Examples
+^^^^^^^^
+
+Execute a pass-through query that selects everything from ``customers`` in
+MySQL::
+
+    SELECT
+      *
+    FROM
+      mysql.tiny.customers(
+        system.query(
+          query => 'SELECT *'
+        )
```

Comment:
The right syntax is
```sql
SELECT * FROM TABLE(system.query(query => 'SELECT * FROM tiny.customers'))
```

Reply:
so how to you specify which catalog to use in that query?

Reply:
```sql
SELECT * FROM table(mysql.system.query(query => '...'));
SELECT * FROM table(postgresql.system.query(query => '...'));
```

Reply:
How do you escape the single quotes in the remote query?

Reply:
Adding extra single quote around each single quote on remote query will do.

`SELECT * FROM TABLE(<catalog>.system.query(query => 'SELECT ''1'' a'))`